runtime/doc/lsp.txt

```diff
@@ -1841,6 +1841,18 @@ typehierarchy({kind})                            *vim.lsp.buf.typehierarchy()*
     Parameters: ~
       • {kind}  (`"subtypes"|"supertypes"`)
 
+workspace_diagnostics({opts})            *vim.lsp.buf.workspace_diagnostics()*
```

Comment:
@gpanders any suggestions on how to document the recommended usage of this function? Not sure if invoking it at regular intervals with a timer would be the best, but idk what event to suggest here either.

Reply:
As a reference from the spec: "In contrast to the document diagnostic request the workspace request can be long running and is not bound to a specific workspace or document state."

Reply:
is that a decision we need to make right now?